### PR TITLE
Add New Featured Artist: Ashrount news post

### PR DESCRIPTION
**Pull request by Ephemeralis**
🌋

So ends the two-per-week release schedule that we've maintained religiously since September 2020. pishi's gone so far above and beyond with all this that it honestly beggars belief.

### CODE REVIEW

File: news/2023/2023-03-25-new-featured-artist-ashrount.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+date: 2023-03-25 11:00:00 +0000
+---
+
+**Ashrount** pounds through the doors to our Featured Artist library.
+
+![](https://assets.ppy.sh/artists/352/header.jpg)
+
+You're more familiar with **Ashrount** than you may think. While their newest productions are under the alias **Ashrount**, they also produced [Emptiness](https://osu.ppy.sh/beatmapsets/1210804) under the name *Xect.* for BOFU2017, and a heap of older songs under the moniker *Maverick*, like the multi-mode marathon set [Maverick - Shironaga Star Jet -aquamarine mix-](https://osu.ppy.sh/beatmapsets/1012634).
+
+Regardless of how you know **Ashrount**, it's clear that their songs are in a league of their own.
```
Comment (Walavouchey) [inline news/2023/2023-03-25-new-featured-artist-ashrount.md:13]:
```suggestion
Regardless of how you may know **Ashrount**, it's clear that their songs are in a league of their own. 
```

File: news/2023/2023-03-25-new-featured-artist-ashrount.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+Try out [the map from the video above](https://osu.ppy.sh/beatmapsets/1955283) hosted by [Niva](https://osu.ppy.sh/users/197805)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/352/Songs/Maverick%20-%20opaque.mp3">
+</audio>
+
+### Ashrount - Cataraxir
+
+Anyone who followed Corsace 2020 would recognize [this map](https://osu.ppy.sh/beatmapsets/1276331) created by [Realazy](https://osu.ppy.sh/users/918297)!
```
Comment (Walavouchey) [inline news/2023/2023-03-25-new-featured-artist-ashrount.md:35]: brilliant, i can't check this due to the website not having past mappools and there being no wiki page either

forum page doesn't even link spreadsheets

beatmap description says "corsace grandfinals - NM3", i'll just trust that this refers to the correct tournament

```suggestion
Anyone who followed [Corsace Open 2020](https://osu.ppy.sh/community/forums/topics/1117976) would recognise [this map](https://osu.ppy.sh/beatmapsets/1276331) created by [Realazy](https://osu.ppy.sh/users/918297)!
```

File: news/2023/2023-03-25-new-featured-artist-ashrount.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/352/Songs/Xect.%20-%20Emptiness.mp3">
+</audio>
+
+---
+
+**Ashrount** has even more top-tier songs to choose from. Take a peek at [**Ashrount**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/352) for the full list, all of which are available as pre-timed beatmap templates!
+
+## Featured Artist release schedule
```
Comment (Walavouchey) [inline news/2023/2023-03-25-new-featured-artist-ashrount.md:69]:
```suggestion
## Featured Artist release schedule update
```

File: news/2023/2023-03-25-new-featured-artist-ashrount.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+With this massive library (which we'd like to remind people is very possibly *the* biggest collection of licensed rhythm gaming music on the planet), the Featured Artist program has already met and gone far beyond its initial goal of providing osu! mappers with a wide variety of songs to choose from.
+
+That library will continue to grow of course, but given its current size, we aim to take less of a *"spam as many artists as possible"* approach, and shift towards **involving the broader osu! community in Featured Artists.**
+
+*Experimentation* is how we'll handle this going forward. We want to avoid complacency with cookie-cutter releases (...like how it's been recently), and instead make Featured Artists more fun by pairing releases with extra projects/events!
+
+So to kick things off, next week's Featured Artist announcement will also have an accompanying livestream on [the osulive Twitch channel](https://www.twitch.tv/osulive) where [pishifat](https://osu.ppy.sh/users/3178418) (a.k.a. the dude writing this) creates a mapset from start to finish using the new artist's music. If you make a map too, it'll be reviewed on the stream as well!
```
Comment (Walavouchey) [inline news/2023/2023-03-25-new-featured-artist-ashrount.md:81]:
```suggestion
So to kick things off, next week's Featured Artist announcement will also have an accompanying livestream on the [osulive Twitch channel](https://www.twitch.tv/osulive) where [pishifat](https://osu.ppy.sh/users/3178418) (a.k.a. the dude writing this) creates a beatmap from start to finish using the new artist's music. If you make a map too, it'll be reviewed on the stream as well!
```

**Review by peppy — APPROVED** [review]

**Review by peppy — DISMISSED** [review]

**Review by peppy — APPROVED** [review]

**Pull request merged**